src/components/DataItem/DataItem.tsx

```diff
@@ -32,7 +42,7 @@ const styles = {
   icon: (bgColor: string) => css`
     background-color: ${bgColor};
     width: 74px;
-    height: 79px;
+    min-height: 79px;
```

Comment:
```suggestion
    min-height: 79px;
    height: 100%;
```

And then we can remove the `<Box display="flex">` wrapping the `img` as it's not needed